lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -15,8 +15,8 @@ def initialize(term_config)
         @term_config = term_config
       end
 
-      attr_reader :term_config, :full_graph, :filtered_graph, :language, :id, :uri, :access_time_s, :normalize_time_s, :fetched_size, :normalized_size, :subauthority
-      private :full_graph, :filtered_graph, :language, :id, :uri, :access_time_s, :normalize_time_s, :fetched_size, :normalized_size, :subauthority
+      attr_reader :term_config, :full_graph, :filtered_graph, :language, :id, :uri, :access_time_s, :normalize_time_s, :subauthority, :request_header
```

Comment:
* Make request_header available throughout.  
* Calculations for fetched_size and normalized size moved to method generating the performance data to return with the results.